fastLatency.py

```diff
@@ -15,8 +15,6 @@
                   help="Specify Output Filename", metavar="filename")
 parser.add_option("--vt1", type="int", dest="vt1",
                   help="VThreshold1 DAC value for all VFATs", metavar="vt1", default=100)
-parser.add_option("--mspl", type="int", dest = "MSPL", default = 4,
```

Comment:
Yes, as I responded above:

> All of the scan scripts had this option hard coded in each of the scan options. So I moved it to qcoptions.py

See lines 3-4 of:

https://github.com/bdorney/vfatqc-python-scripts/blob/develop/qcoptions.py